playground/App.vue

```diff
@@ -28,17 +30,17 @@
     <div class="carousel-wrapper">
       <VueCarousel
         v-model="currentSlide"
-        :items-to-show="parseFloat(itemsToShow)"
-        :items-to-scroll="parseFloat(itemsToScroll)"
+        :items-to-show="itemsToShow"
+        :items-to-scroll="itemsToScroll"
         :gap="10"
-        :height="height || 'auto'"
+        :height="parseInt(height) || 'auto'"
```

Comment:
I added something to handle this correctly on the component side lower (|| parseInt(config.height) == config.height),


This allows for other values like `30vh` or `10%`
though why remove the parseFloat on itemsToShow and toScroll?